--- /tmp/monkey_patches.rb	2018-10-01 22:41:24.753411959 -0700
+++ /var/lib/gems/2.5.0/gems/puppet-3.8.7/lib/puppet/util/monkey_patches.rb	2018-10-01 22:42:15.485207959 -0700
@@ -175,7 +175,9 @@
   else
     DEFAULT_PARAMS[:options] = OpenSSL::SSL::OP_NO_SSLv2 | OpenSSL::SSL::OP_NO_SSLv3
   end
-  DEFAULT_PARAMS[:ciphers] << ':!SSLv2'
+  if DEFAULT_PARAMS[:ciphers]
+    DEFAULT_PARAMS[:ciphers] << ':!SSLv2'
+  end
 
   alias __original_initialize initialize
   private :__original_initialize
